test/commands/force/data/bulk/status.test.ts

```diff
@@ -4,136 +4,196 @@
+    },
+    check(): Promise<JobInfo> {
+      throw new Error('Function not implemented.');
+    },
+    ready(): Promise<string> {
+      throw new Error('Function not implemented.');
+    },
+    close(): Promise<JobInfo> {
+      throw new Error('Function not implemented.');
+    },
+    abort(): Promise<JobInfo> {
+      throw new Error('Function not implemented.');
+    },
+    _changeState(state: JobState): Promise<JobInfo> {
+      throw new Error('Function not implemented.');
+    },
+    addListener(eventName: string | symbol, listener: (...args: any[]) => void): Job<any, BulkOperation> {
+      throw new Error('Function not implemented.');
+    },
+    on(eventName: string | symbol, listener: (...args: any[]) => void): Job<any, BulkOperation> {
+      throw new Error('Function not implemented.');
+    },
+    once(eventName: string | symbol, listener: (...args: any[]) => void): Job<any, BulkOperation> {
+      throw new Error('Function not implemented.');
+    },
+    removeListener(eventName: string | symbol, listener: (...args: any[]) => void): Job<any, BulkOperation> {
+      throw new Error('Function not implemented.');
+    },
+    off(eventName: string | symbol, listener: (...args: any[]) => void): Job<any, BulkOperation> {
+      throw new Error('Function not implemented.');
+    },
+    removeAllListeners(event?: string | symbol): Job<any, BulkOperation> {
+      throw new Error('Function not implemented.');
+    },
+    setMaxListeners(n: number): Job<any, BulkOperation> {
+      throw new Error('Function not implemented.');
+    },
+    getMaxListeners(): number {
+      throw new Error('Function not implemented.');
+    },
+    // eslint-disable-next-line @typescript-eslint/ban-types
+    listeners(eventName: string | symbol): Function[] {
+      throw new Error('Function not implemented.');
+    },
+    // eslint-disable-next-line @typescript-eslint/ban-types
+    rawListeners(eventName: string | symbol): Function[] {
+      throw new Error('Function not implemented.');
+    },
+    emit(eventName: string | symbol, ...args: any[]): boolean {
+      throw new Error('Function not implemented.');
+    },
+    listenerCount(eventName: string | symbol): number {
+      throw new Error('Function not implemented.');
+    },
+    prependListener(eventName: string | symbol, listener: (...args: any[]) => void): Job<any, BulkOperation> {
+      throw new Error('Function not implemented.');
+    },
+    prependOnceListener(eventName: string | symbol, listener: (...args: any[]) => void): Job<any, BulkOperation> {
+      throw new Error('Function not implemented.');
+    },
+    eventNames(): Array<string | symbol> {
+      throw new Error('Function not implemented.');
+    },
+  };
+  beforeEach(async () => {
+    await $$.stubAuths(testOrg);
+    await config.load();
+    stdoutSpy = $$.SANDBOX.stub(process.stdout, 'write');
+
+    $$.SANDBOXES.CONNECTION.stub(Bulk.prototype, 'job').returns({
```

Comment:
I _think_ `stubMethod` from @salseforce/ts-types lets you create stubs like this without need to add in `...unusedInterfaceParts`

Reply:
pretty sure I tried that for a bit and gave up.  Wasn't sure from the other examples of it in use how to make it work with testSetup, and then the types still weren't right.